weasyl/forms.py

```diff
@@ -6,15 +6,16 @@
 _T = TypeVar("T")
 
 
-def expect_id(s: str) -> int:
+def expect_id(s: str, error_message="Unexpected") -> int:
```

Comment:
No changes to this function, please. Keep it simple. `isinstance(s, str)` shouldn’t be checked on `s: str`.

Reply:
Will revert this.